packages/venia-concept/src/RootComponents/Category/categoryContent.js

```diff
@@ -3,6 +3,7 @@ import classify from 'src/classify';
 import Gallery from 'src/components/Gallery';
 import Pagination from 'src/components/Pagination';
 import defaultClasses from './category.css';
+import { RouteConsumer } from '@magento/peregrine';
```

Comment:
I believe a standard `Route` from `react-router-dom` should work here, since we're on `4.4.x`.

Reply:
shall we use withRouter API, I have made the implementation and commited the files